src/shop/tests/test_api.py

```diff
@@ -27,9 +32,13 @@ def parse_delivery_report(self, report_path):
     def retrieve_product(self, sku):
         return SupplierProduct(
             name='Billys Original',
-            price=Decimal('9.25')
+            price=Decimal('9.25'),
+            units=1
         )
 
+    def order_product(self, sku, qty):
+        pass
```

Comment:
`order_product` is tested using another mocking method and we could probably do the same for `retrieve_product` and `parse_delivery_report`. Then, that dummy class could be removed entirely.